src/styles/custom/_navbar.scss

```diff
@@ -277,4 +285,82 @@
 
 }
 
+.navbar-nav {
+  > li > .dropdown-menu, {
+    background-color: $brand-primary;
+  }
+
+  .dropdown-menu {
+    border: none;
+    border-radius: 0 0 5px 5px;
+    > li:not(.divider) {
+      padding: $dropdownMenu-content 0;
+      margin: 0 ($dropdownMenu-content * 2);
+
+      > a {
+        color: $white;
+        font-weight: 700;
+        &:hover,
+        &:focus {
+          background-color: $white;
+          color: $brand-primary;
+        }
+      }
+    }
+  }
+
+  &.navbar-right {
+    .dropdown-menu {
+      li:first-child {
+        margin: 0 ($dropdownMenu-content * 2);
+        border-top: 1px solid #aaa;
+        border-bottom: 1px solid #aaa;
+
+        p {
+          margin: 0;
+          padding: 0;
+          color: $white;
+          font-style: italic;
+          font-weight: 400;
+
+          &.label {
+            font-size: 80%;
+          }
+        }
+
+      }
+
+      li:last-child {
+        padding: $dropdownMenu-content 0 ($dropdownMenu-content * 2) 0;
+        margin: 0 ($dropdownMenu-content * 2);
+
+        .btn {
```

Comment:
Not sure if this belongs to cbp-theme or the cf-menu-js

Reply:
Belongs to cbp-theme